zilliqa/src/cfg.rs

```diff
@@ -568,3 +610,130 @@ impl Default for ContractUpgradesBlockHeights {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_get_forks_with_no_forks() {
+        let config = ConsensusConfig {
+            genesis_fork: genesis_fork_default(),
+            forks: vec![],
+            ..Default::default()
+        };
+
+        let forks = config.get_forks().unwrap();
+        assert_eq!(forks.0.len(), 1);
+        assert_eq!(forks.0[0].at_height, 0);
+    }
+
+    #[test]
+    fn test_get_forks_with_one_fork() {
+        let config = ConsensusConfig {
+            genesis_fork: genesis_fork_default(),
+            forks: vec![ForkDelta {
+                at_height: 10,
+                failed_scilla_call_from_gas_exempt_caller_causes_revert: None,
+                call_mode_1_sets_caller_to_parent_caller: Some(true),
+                scilla_messages_can_call_evm_contracts: None,
+                scilla_contract_creation_increments_account_balance: Some(true),
+            }],
+            ..Default::default()
+        };
+
+        let forks = config.get_forks().unwrap();
+        assert_eq!(forks.0.len(), 2);
+        assert_eq!(forks.0[0].at_height, 0);
+        assert_eq!(forks.0[1].at_height, 10);
```

Comment:
I think these tests should be using `Forks::get` instead to get the fork at a specific block height.

That also gives us the chance to test the boundary cases to ensure we don't have any off by one errors (I was worried about this when implementing `get`)